pages/_app.js

```diff
@@ -64,6 +63,15 @@ function MyApp({ Component, pageProps }) {
     event.target.reset();
     handleIsCreating();
   }
+
+  async function handleDeleteMemoryCard(id) {
+    await fetch("/api/memories/" + id, {
+      method: "DELETE",
+    });
+
+    getMemories();
+  }
```

Comment:
das ist etwas anders geschrieben, als ich es in meinen Notizen hab, aber hey, wenn es funktioniert, dann ist es prima :)

Reply:
Aus Interesse, was hast du dir hier anderes notiert? 🤔

Reply:
ich glaube das trifft in deinem Fall nicht zu, aber für die Datenbank dann
if (req.method === 'DELETE') {const id = req.query.id; await Question.findByIdAndDelete(id); res.status etc etc 

aber das wäre einen Schritt weiter oder?